src/components/Table.tsx

```diff
@@ -41,13 +41,15 @@ interface TableCellProps {
 const Table: FC<TableProps> = ({ className, children, headers, isLoading }) => (
   <ScrollableWrapper>
     <div className={classNames(className, { 'skeleton-loader': isLoading })}>
-      <TableHeaderRow columnWidths={headers.map(({ width }) => width)}>
-        {headers.map(({ title, align }) => (
-          <TableHeaderCell key={title} align={align}>
-            {title}
-          </TableHeaderCell>
-        ))}
-      </TableHeaderRow>
+      {headers && headers.length > 0 && (
+        <TableHeaderRow columnWidths={headers.map(({ width }) => width)}>
+          {headers.map(({ title, align }) => (
+            <TableHeaderCell key={title} align={align}>
+              {title}
```

Comment:
Is this translated?

Reply:
Any titles that are passed in will have to be. The component itself shouldn't be responsible for this, but rather the user that is passing in the title (like you commented above, where I'll now need to remove translations since I created them in that particular file to be passed into this component).